docs/api_familie_klantinteracties/klantinteracties_CRUD/openapi.yaml

```diff
@@ -4189,116 +4204,33 @@ components:
           $ref: "#/components/schemas/Adresregel"
         adresregel3:
           $ref: "#/components/schemas/Adresregel"
-        land:
-          $ref: "#/components/schemas/Land"
+        codeLand:
+          $ref: "#/components/schemas/Identificatiecode"
           description: "Een code, opgenomen in Tabel 34 Landentabel, die een land\
-              \ buiten Nederland aangeeft waar het adres gevonden kan worden."
+              \ buiten Nederland aangeeft waar het adres gevonden kan worden. (Referntielijst Land)"
         nummeraanduidingid:
           $ref: "#/components/schemas/Identificatiecode"
-    ExternRegister:
-      type: "object"
-      description: "Lijst van registers buiten het domein van klantinteracties waarin\
-        \ objecten zijn geregistreerd die binnen het domein van klantinteracties een\
-        \ rol (kunnen) spelen."
-      required:
-      - "code"
-      - "locatie"
-      - "naam"
-      properties:
-        code:
-          $ref: "#/components/schemas/Identificatiecode"
-        locatie:
-          $ref: "#/components/schemas/Omschrijving-kort"
-          readOnly: true
-        naam:
-          $ref: "#/components/schemas/Naam-kort"
-          readOnly: true
     Identificator:
```

Comment:
Vraag over referentielijsten voor objectidentificatoren. Ik had gedacht (maar blijkbaar niet uitgesproken) dat zo'n referentielijst (hoewel het dan stikt genomen niet meer helemaal een lijst is) op iedere regel een een geldige combinatie van register, objecttype en soortObjectId zou omvatten. Maar ik realiseer me nu pas dat we het helemaal niet zo hebben vormgegeven. Zou het niet toch te overwegen zijn alsnog deze weg - die voorkomt dat per ongeluk 'ongeldige combinaties worden geregistreerd van bijvoorbeeld objecttypen met soortenID's die helemaal niet voorkomen in de registers waaraan ze gekoppeld zijn geraakt - te volgen?

Bijvoorbeeld :
```
| BRP                              | Natuurlijk Persoon | BSN                  |
| Zakenregister VTH                | Zaak               | (zaak)identificatie  |
| Centraal Klantinteractieregister | Partij             | (partij)nummer       |
| Centraal Klantinteractieregister | Klantcontact       | (klantcontact)nummer |
```

Reply:
> Hoe zie je die validatie dan voor je?

Ik bedoelde met 'geldige combinatie' niet per se 'gevalideerde combinatie', maar ga ervan uit dat je bij het configureren van een referentielijst iets beter oplet voor je een nieuwe waarde toevoegt dan wanneer je - gechargeerd, want de consumer kan hier natuurlijk ook bij helpen - voor de 23e keer op één dag de juiste combinatie van register, objecttype en soortObjectId bij elkaar puzzelt.

> Moet een referentielijst voor register dan een een volledige set aan alle mogelijke objecttypen bevatten, en dan per objecttype ook weer een lijst van alle attributen (=soortObjectId)?

Ik neem juist aan dat je per register alleen de objecttypen opneemt waarnaar je in redelijkheid zou kunnen willen verwijzen. Dat zal een beperkt aantal zijn. Hoe beperkt zal de praktijk moeten uitwijzen. Voor soortObjectId geldt hetzelfde, maar nog sterker; ik kan me bijna niet voorstellen dat er gevallen zijn waar je meer dan twee attributen van één object als `objectId` wil kunnen overnemen.